integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXUpgradeTests.java

```diff
@@ -0,0 +1,130 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.distrox;
+
+import static com.sequenceiq.it.cloudbreak.cloud.HostGroupType.COMPUTE;
+import static com.sequenceiq.it.cloudbreak.cloud.HostGroupType.MASTER;
+import static com.sequenceiq.it.cloudbreak.cloud.HostGroupType.WORKER;
+import static com.sequenceiq.it.cloudbreak.context.RunningParameter.key;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.testng.annotations.Test;
+
+import com.sequenceiq.it.cloudbreak.client.DistroXTestClient;
+import com.sequenceiq.it.cloudbreak.client.SdxTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.dto.distrox.cluster.DistroXUpgradeTestDto;
+import com.sequenceiq.it.cloudbreak.dto.sdx.SdxTestDto;
+import com.sequenceiq.it.cloudbreak.dto.sdx.SdxUpgradeTestDto;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest;
+import com.sequenceiq.it.cloudbreak.util.CloudFunctionality;
+import com.sequenceiq.it.cloudbreak.util.DistroxUtil;
+import com.sequenceiq.it.cloudbreak.util.InstanceUtil;
+import com.sequenceiq.it.cloudbreak.util.VolumeUtils;
+import com.sequenceiq.it.cloudbreak.util.spot.UseSpotInstances;
+import com.sequenceiq.sdx.api.model.SdxClusterStatusResponse;
+import com.sequenceiq.sdx.api.model.SdxUpgradeReplaceVms;
+
+public class DistroXUpgradeTests extends AbstractE2ETest {
+
+    @Inject
+    private SdxTestClient sdxTestClient;
+
+    @Inject
+    private DistroXTestClient distroXTestClient;
+
+    @Inject
+    private DistroxUtil distroxUtil;
+
+    private String distroXName;
+
+    private List<String> expectedVolumeIds = new ArrayList<>();
+
+    @Override
+    protected void setupTest(TestContext testContext) {
+        createDefaultUser(testContext);
+        createDefaultCredential(testContext);
+        initializeDefaultBlueprints(testContext);
+        createEnvironmentWithNetworkAndFreeIpa(testContext);
+
+        String sdxName = resourcePropertyProvider().getName();
+        distroXName = resourcePropertyProvider().getName();
+        testContext
+                .given(sdxName, SdxTestDto.class)
+                .withCloudStorage()
+                .withRuntimeVersion("7.2.7")
```

Comment:
I would use the `application.yml` and a related environment variable on Jenkins to parameterise this runtime version. If the version is going to be changed we do not need an additional commit to update the test as well:
```
  runtimeVersion: 7.2.8
  upgrade:
    currentRuntimeVersion: 7.2.0
    targetRuntimeVersion: 7.2.8
```
at latest [application.yml](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/resources/application.yml#L78-L82)